includes/controllers/AuthController.php

```diff
@@ -135,29 +140,20 @@ public function checkPasswordValidateRequirements(string $password): bool
      */
     public function connectUser()
     {
-        $userFromSession = $this->getLoggedUser();
-        if (!empty($userFromSession['name'])) {
-            // check if user ever existing
-            $user = $this->userManager->getOneByName($userFromSession['name']);
-            $remember = $_SESSION['user']['remember'] ?? 0;
-            if (!empty($user)) {
-                if (empty($_SESSION['user']['lastConnection'])) {
-                    if (!$this->wiki->UserIsAdmin()) {
-                        // do not disconnect admin during update
-                        $user = null;
-                    }
-                } elseif ((intval($_SESSION['user']['lastConnection']) + ($remember ? 90 * 24 * 60 * 60 : 60 * 60)) < time()) {
-                    // like Session.class->setPersistentCookie()
-                    // If $remember is set and different from 0, 90 days, 1 hour otherwise
-                    $user = null;
-                }
+        $this->cleanOldFormatCookie();
```

Comment:
est ce nécessaire de faire le ménage à chaque login? C'est pas cela qui justement fait des erreurs type `Cookie “name” has been rejected because it is already expired. `

Reply:
Oui c'est nécessaire pour éviter de garder un vieux cookie qui contient un md5 du mot de passe.

En plus, `$this->cleanOldFormatCookie(` (https://github.com/YesWiki/yeswiki/pull/1046/files/e02e225a1cfd6f7cf4d3cbeb30cf7658cdb325e0#diff-04c21f144fdfe1c68db0bf4c426ed68b4d1dbaa3ea393c646ec1dccc93685749R382-R393) ne supprime les cookies que s'ils sont définis, ce qui va retirer le warning intempestif actuel